tests/core/test_invite.py

```diff
@@ -383,6 +384,11 @@ async def _do_claimer():
                 with pytest.raises(InvitePeerResetError):
                     await greeter_in_progress_ctx.do_wait_peer_trust()
 
+                if OXIDIZED:
+                    # Reinitialize greeter_initial_ctx because it has beed consumed
+                    greeter_initial_ctx = UserGreetInitialCtx(
```

Comment:
This seems pretty cumbersome, why not instead have the python binding of `UserGreetInitialCtx` first do a clone on it internal before calling the method on it ?

Reply:
Done